pb-rs/src/types.rs

```diff
@@ -312,7 +312,7 @@ impl FieldType {
                 format!("{}{}{}", m.get_modules(desc), m.name, lifetime)
             }
             FieldType::Map(ref key, ref value) => format!(
-                "HashMap<{}, {}>",
+                "KVMap<{}, {}>",
```

Comment:
```suggestion
                "{}<{}, {}>",
                    if config.no_std { "KVMap" } else { "HashMap" },
```

Reply:
Hey @tafia, sorry for the delay in the reply! Thanks for taking a look :smile: I added the `KVMap` because the `rust_type` function doesn't have access to the config object (and I didn't want to change the call definition for a bunch of function), and the alias shouldn't restrict with what the actual person uses (other than reflecting in the docs). Obviously, I'd be happy to change it if you want, but I wanted to check with you first. I could of course also alias `BTreeMap` to `HashMap` but that seems a little hacky...

Reply:
I checked, the alias, actually does not reflect in the docs because its private. It's really pretty invisible to the user of the struct...